addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/CloudMetadataScanRuleUnitTest.java

```diff
@@ -63,11 +63,17 @@ void shouldNotAlertIfResponseIsNot200Ok() throws Exception {
         rule.scan();
         // Then
         assertThat(alertsRaised, hasSize(0));
-        assertEquals(2, httpMessagesSent.size());
+        assertEquals(4, httpMessagesSent.size());
```

Comment:
Since the actual number does not matter (IMO) it could just check it's not empty.